javascript/javascript.js

```diff
@@ -0,0 +1,31 @@
+/* global $ */
+
+$(document).ready(function () {
+  $('a').click(function (event) {
```

Comment:
Just so you know, this will prevent ALL links from working as intended on the page. It would have been best to add this for each individual function, but in the context of this assignment, we don't care about the other links. Just be careful with global declarations like this.

Reply:
Ahh!  I figured this out in class when he was adding it for each individual function.  I was like “ohhh...dammit!”

I’ll know this for next time!